main.go

```diff
@@ -31,8 +31,10 @@ import (
 	"github.com/spf13/pflag"
```

Comment:
One additional thing, I forgot to mention before.

Please go through the corresponding manager.yaml's of all controllers and add the following to args: `,LazyRestmapper=${EXP_LAZY_RESTMAPPER:=false}`. 

This makes it easy to enable the feature gate by just setting the env var before running clusterctl init / or in tilt-settings.yaml for dev